### ISSUE TITLE
Cyber-security bundle

### ISSUE THREAD

**Issue by oalam** [Hurence/logisland#119]
a logisland bundle is a package of plugins/conf & visuals that address a specific use case, here we talk about threats and cyber security.

**Comment by waschult** [comment]
Have a look at Apache Spot. Something like reputation database (e.g. firehol) would be nice.

**Comment by oalam** [comment]
you're absolutely right. We'll think how we could integrate with this framework. thanks
